File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/R. Age in Days/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/R
#include<iostream>
using namespace std;
 
int main(){
    unsigned int n;
    cin >> n;
    cout << n/365 << " years\n";
    cout << (n%=365)/30 << " months\n";
    cout << (n%=30) << " days\n";
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/R. Age in Days/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/R

using System;

namespace Codeforces.Assuit.Sheet1
{
    class ProblemR
    {
        static void Main(string[] args)
        {
            uint n = uint.Parse(Console.ReadLine());
            Console.WriteLine($"{n/365} years");
            Console.WriteLine($"{(n%=365)/30} months");
            Console.WriteLine($"{n%=30} days");
        }
    }
}
